node-aws-lambda/README.md

```diff
@@ -8,11 +8,22 @@ Before following this guide we recommend reading the basic [Node + Rookout] guid
 
 There are 3 simple steps to integrate Rookout into your existing Node application:
 
-1. Installing the Rookout SDK: `npm install --save rookout` and adding it in the entry file `const rookout = require('rookout/lambda');`
+1. Add our lambda layer, you can find the latest version [here](https://docs.rookout.com/docs/sdk-digests.html), make sure to replace the region and version placeholder with yours  
```

Comment:
@MickaelAlliel @BenTheKlein can we somehow make it more updated? Maybe some badge?

Reply:
A badge would require some work, AWS doesn't provide them.
the arn looks like `arn:aws:lambda:<YOUR_REGION>:032<YOUR_RUNTIME>5105219:layer:rookout_python<YOUR_RUNTIME>_v_0_1_74_1:1` and changes each version. No easy way to have it updated in the example here